com.github.Qv2ray.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
+    "cleanup": [
+        "/include",
+        "/lib/debug",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=~/.qv2ray",
```

Comment:
I think you missed my comment about this: 

> Well, you can store user settings by using https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html. I just don't see why you want those settings to be in $HOME/.config/qv2ray or $HOME/.qv2ray instead of just $XDG_CONFIG_HOME/qv2ray/
By using that, the settings will be stored under $HOME/.var/app/com.github.lhy0403.Qv2ray/config/qv2ray under Flatpak and in $HOME/.config/qv2ray otherwise.

Reply:
Like this?

Reply:
or maybe I should remove ```"--filesystem=~/.qv2ray",``` too?

Reply:
This has to be fixed from the application and both permissions could be removed once it's done

Reply:
I got it thanks.

Reply:
I add a patch to do that. It should be work.

Reply:
Let me try this.

Reply:
Work perfectly. And we are already fixed [it](https://github.com/Qv2ray/Qv2ray/commit/d601d2ee1216cd5f09e90c7c627f39326e8de936).